static/js/vue-cdr-access/src/components/facets.vue

```diff
@@ -240,17 +245,17 @@
              * @returns {string}
              */
             facetValue(value) {
-                let facet_type = this.facetType(value);
-
-                const current_facet_value = this.selected_facets.filter((f) => {
-                    const regex = new RegExp(facet_type);
-                    return regex.test(f);
-                });
+                const facet_type = this.facetType(value);
+                const current_facet_value = this.selected_facets.filter(f => f.startsWith(facet_type));
 
                 if (current_facet_value.length === 1) {
-                    const joined_facets = `${current_facet_value[0]}||${value.limitToValue}`;
-                    const deduped_facets = Array.from(new Set(joined_facets.split('||')));
-                    return `${deduped_facets.join('||')}`
+                    const selected_facet_parts = current_facet_value[0]
+                        .replace(new RegExp(facet_type), '')
```

Comment:
I don't think the regexp is needed here, replacing `facet_type` by itself should work?